File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
C = int(input())

for i in range(C):
    data =list(map(int,input().split()))
    S = data[0] 
    A = data[1:]
    avg = 0
    for j in range(S):
        if sum(A)/S < A[j]:
            avg += 1
    M = avg/S * 100
    print(f'{M:.3f}%')
    
    

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.kt
import java.util.Scanner

fun main() = with(Scanner(System.`in`)){
    val caseSize = nextInt()
    
    repeat(caseSize) {
        val students = nextInt()
        val studentsGrades = Array(students) { 0 }
        
        for ( i in 0 until students) {
            studentsGrades[i] = nextInt()
        }
        
        val avg = studentsGrades.sum() / students
        val overAvg = studentsGrades.filter { it > avg}.count()
        val result = String.format("%.3f", (overAvg.toFloat() / students * 100).toDouble()) + "%"
        
        println(result)
    }
}